src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildANetCoreApp.cs

```diff
@@ -137,6 +137,8 @@ public void The_RuntimeFrameworkVersion_can_float()
             var testAsset = _testAssetsManager.CreateTestProject(testProject, testIdentifier)
                 .Restore(Log, testProject.Name, extraArgs);
 
+            NuGetConfigWriter.Write(testAsset.TestRoot, NuGetConfigWriter.DotnetCoreMyGetFeed);
```

Comment:
@dsplaisted  similar to https://github.com/dotnet/cli/pull/9507 need to scope the feed

Reply:
Is there any guarantee that all of the packages will be published to this feed?  The blob storage feed was supposed to be the "One feed to rule them all", so I'm worried that we could end up not picking up a new patch because it wasn't published to the right feed.

@johnbeisner